File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.cpp
class Solution {
public:
    bool arrayStringsAreEqual(vector<string>& word1, vector<string>& word2) {
        string res="", ans="";
        for(auto s:word1)
            res += s;
        for(auto v:word2)
            ans += v;
        return ans==res;
    }
};

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        String res="", ans="";
        for(String s:word1)
            res += s;
        for(String s:word2)
            ans += s;
        return res.equals(ans);
    }
}